### DISCUSSION TITLE
Integrazione gestionali via API

### DISCUSSION THREAD

**Discussion by IvanRevelli** [fattureincloud/api#436] [Q&A] [answered]
Buongiorno, un nostro cliente che al momento utilizza fatture in cloud solo per la gestione delle fe, avrebbe l'esigenza di generare gli xml delle fe con un'altro gestionale (causa loro esigenze di gestione della produzione). Non ho trovato metodi REST che consentono di fare upload di un xml fe e nemmeno di scaricare gli xml fe dei ricevute tramite il vostro servizio, mi confermate che l'unico metodo per creare una fe tramite le vs api è creare un json come request del metodo "/c/{company_id}/issued_documents"?

**Comment by valmoz** [comment, marked as answer]
Buongiorno @IvanRevelli 
confermo. Le nostre API non consentono l'import di XML generati esternamente.

Grazie
Mauro

**Reply by IvanRevelli** [reply]
Grazie per la celere risposta.
Buona giornata
